src/components/elementView/elementTable.tsx

```diff
@@ -863,44 +952,44 @@ class ElementTable extends React.Component<
                         })}
                         <td>
                           {this.props.isAdmin &&
-                          this.props.type !== ElementType.USER ? (
-                            <div className="inline-buttons">
-                              <AnchorButton
-                                className="button-table"
-                                intent="primary"
-                                icon="edit"
-                                minimal
-                                onClick={(event: any) => {
-                                  this.handleEditButtonClick(item);
-                                  event.stopPropagation();
-                                }}
-                              />
-                              <AnchorButton
-                                className="button-table"
-                                intent="danger"
-                                minimal
-                                icon="trash"
-                                onClick={(event: any) => {
-                                  this.handleDeleteButtonClick(item);
-                                  event.stopPropagation();
-                                }}
-                              />
-                              {this.props.isAdmin &&
-                              isAssetObject(item) &&
-                              item.rack.is_network_controlled ? (
+                            this.props.type !== ElementType.USER ? (
+                              <div className="inline-buttons">
```

Comment:
I don't know why git thinks I changed all the code around here (maybe automatic formatting changes???) but I don't think any of the code beyond this point actually changed